app/assets/stylesheets/style.scss

```diff
@@ -18,22 +18,22 @@ body {
   background: #FAFAFA;
   // background: radial-gradient(ellipse at center, $color_steel_gray_approx 0, $color_hunter_green_approx 100%);
   color: $blue;
- a{
-  color:$blue;
-  font-family: $font_0;
- }
+  a{
+    color:$blue;
+    font-family: $font_0;
+  }
 
- h1, h2, h3, h4, h5, h6{
-  color:$blue;
-  opacity: 0.8;
-  font-family: $font_0;
- }
+  h1, h2, h3, h4, h5, h6{
```

Comment:
Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space